src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styles from './DropdownMenu.module.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faIdCard, faSignOutAlt } from '@fortawesome/free-solid-svg-icons';
+import { useNavigate } from 'react-router';
+import { useTranslation } from 'react-i18next';
+
+type Props = {
+    setOpenMenu: (openMenu: boolean) => void;
+};
+
+const DropdownMenu: React.FC<Props> = ({ setOpenMenu }) => {
+    const navigate = useNavigate();
+    const { t } = useTranslation();
+
+    const logout = () => {
```

Comment:
Far as I can see it all looks god, just wondering is it better practice if this is in return. Found it somewhere so just a thought. 
`const { t } = useTranslation();` on a line 27.

Reply:
You cannot place hooks in return.

Reply:
you are right, I actually switched it in my head with render